mattermost/mattermost-webapp
# [MM-48771] Mentions fixes

#### Summary

Fixes the text color of highlighted mentions that are not at-mentions. 
Also fixes a bug where clicking on at-mentions in post preview sends the message.

#### Ticket Link

https://mattermost.atlassian.net/browse/MM-48771
https://mattermost.atlassian.net/browse/MM-48925

#### Screenshots

Before:
<img width="470" alt="Screen Shot 2022-12-06 at 1 28 51 PM" src="https://user-images.githubusercontent.com/112951043/205992962-d43e519e-efba-4146-a5f5-6b0792632512.png">

After:
<img width="470" alt="Screen Shot 2022-12-06 at 1 29 20 PM" src="https://user-images.githubusercontent.com/112951043/205992967-db442cb2-0df1-4a02-98d1-68f82842eff0.png">

The group mention has always looked different, it seems to be intentional. 

#### QA Test Steps:

1. Ensure that the highlighted keyword mentions and highlighted first name mentions have the same styling as highlighted at-mentions using custom themes (see screenshots above). 
2. Make a draft message with an at-mention but don't send it.
3. View the preview of the message. Click on the at-mention that renders. 
4. Ensure that the message does not send when clicking on the at-mention.

#### Release Note

```release-note
NONE
```
